modules/jaeger-config-collector.adoc

```diff
@@ -0,0 +1,38 @@
+////
+This REFERENCE module included in the following assemblies:
+-rhbjaeger-deploy.adoc
+////
+
+[id="jaeger-config-collector_{context}"]
+= Jaeger Collector configuration options
+
```

Comment:
Just wondering best way to structure the component configs - because they are different depending on the deployment strategy being used.

So maybe rather than explaining the components individually, explain the relevant components in the context of the chosen deployment strategy?

Reply:
This is exactly what I've been struggling with for the past couple of week.